pythonforandroid/bootstraps/pygame/build/buildlib/argparse.py

```diff
@@ -1468,7 +1468,7 @@ def _check_conflict(self, action):
     def _handle_conflict_error(self, action, conflicting_actions):
         message = _('conflicting option string(s): %s')
         conflict_string = ', '.join([option_string
-                                     for option_string, action
+                                     for option_string, action  # noqa F812

```

Comment:
Looks like Travis is complaining about this in the previous build even though it's OK on my dev instance.
This file will be dropped with https://github.com/kivy/python-for-android/pull/1670 anyway

Reply:
Because it is redefining parameter `action` from the function. This needs to be fixed and not silenced if the content of the file is only moving somewhere else and is not deleted completely.